public/javascripts/redirect.js

```diff
@@ -0,0 +1,74 @@
+// Redirect all the existing URLs with fragment identifiers in to the new section
+(function () {
+  'use strict';
+
+
+  var getNewRouteFor = function (fragment) {
+    var baseURL = 'http://govuk-elements.herokuapp.com';
```

Comment:
For the record, my original suggestion was incorrect as it would result in the inclusion of the pathname too. Thanks for the correction @joelanman.